src/main/java/redis/clients/jedis/commands/MultiKeyBinaryJedisClusterCommands.java

```diff
@@ -31,15 +30,19 @@
 
   byte[] lmove(byte[] srcKey, byte[] dstKey, ListDirection from, ListDirection to);
 
-  byte[] blmove(byte[] srcKey, byte[] dstKey, ListDirection from, ListDirection to, int timeout);
+  byte[] blmove(byte[] srcKey, byte[] dstKey, ListDirection from, ListDirection to, double timeout);
 
   List<byte[]> blpop(int timeout, byte[]... keys);
 
+  List<byte[]> blpop(double timeout, byte[]... keys);
+
   List<byte[]> brpop(int timeout, byte[]... keys);
 
-  KeyedTuple bzpopmax(int timeout, byte[]... keys);
+  List<byte[]> brpop(double timeout, byte[]... keys);
```

Comment:
@sazzad16 I think you should deprecate the interface methods and not only the implementation methods

Reply:
@gkorland The deprecated methods were not part of any interface.